pallets/corporate-actions/src/distribution/mod.rs

```diff
@@ -349,12 +349,20 @@ decl_module! {
         #[weight = <T as Trait>::DistWeightInfo::reclaim()]
         pub fn reclaim(origin, ca_id: CAId) {
             // Ensure DID is the dist creator, they haven't reclaimed, and that expiry has passed.
-            let did = <Identity<T>>::ensure_perms(origin)?;
+            let did = <Identity<T>>::ensure_perms(origin.clone())?;
             let dist = Self::ensure_distribution_exists(ca_id)?;
             ensure!(did == dist.from.did, Error::<T>::NotDistributionCreator);
```

Comment:
This line imposes a check that is too strict. Removing this line in favor of the new checks is what is meant by "relax".